inputJsonFiles/Architecture/arch_1PCSIM6WorkerMNist.json

```diff
@@ -6,13 +6,13 @@
   },
   "devices": [
     {
-      "host": "192.168.0.108",
+      "host": "192.168.64.7",

```

Comment:
@halfway258 Please approve this change 
@GuyPerets106  I think it's better approach to duplicate the architecture and add some suffix such as:
arch_1PCSIM6WorkerMNist_lab_network.json